app.js

```diff
@@ -15,14 +15,14 @@ $(document).ready(function() {
     dataType: "json",
     data: {
 
-    }
+    },
     success: searchAnimals
   }); //end ajax object
```

Comment:
This AJAX call needs to happen on button click. Right now, it is running every time you load the page.

You need to put this AJAX call inside the callback function for the button click.